app/src/test/java/com/nervousfish/nervousfish/modules/constants/ConstantsTest.java

```diff
@@ -0,0 +1,40 @@
+package com.nervousfish.nervousfish.modules.constants;
+
+import com.nervousfish.nervousfish.BaseTest;
+import com.nervousfish.nervousfish.service_locator.IServiceLocator;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+
+/**
+ * ----------- NOTE ------------
+ * We cannot test much more than just that the results are not null.
+ * If we change the names of the
```

Comment:
I think you should use JavaDoc comments (`/** */`) for "notes"... Can you remove the `----------- NOTE ------------` and replace the `/** */` with normal comments (`//`)?